### PR TITLE
better doc for lookup/resolution policy, reference policy, and pullthrough

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@dmage @legionus @miminar @csrwng @smarterclayton ptal.

**Comment by smarterclayton** [comment]
Clear and concise, lgtm (but if others want to review more please)

**Comment by csrwng** [comment]
lgtm

**Comment by bparees** [comment]
would like one sign off from one of our registry gurus ( @dmage @legionus @miminar ) and then i'll send it to the docs team.

File: architecture/core_concepts/containers_and_images.adoc [hunk 1]
```diff
@@ -109,9 +109,9 @@ The tag is unique and does not change. When using this tag, the image does not u
 
 [[docker-registries]]
 
-== Container Registries
+== Image Registries
 
-A container registry is a service for storing and retrieving Docker-formatted
+An image registry is a service for storing and retrieving Docker-formatted
```
[the remark this answers is not in the document]
Comment (bparees) [reply architecture/core_concepts/containers_and_images.adoc]: I don't think there was a purpose(or not a good one), i think it was a mistake/misunderstanding that was propagated throughout the docs.  (Consider that we are calling our new repo "openshift/image-registry" and not "openshift/container-registry")

That said, we now call it the redhat container catalog, so the mistake has propagated all the way to the top so i guess i'll undo this, but i'm not thrilled about it.  (containers!=images).

At some point I think this should be revisited.

[the remark this answers is not in the document]
Comment (bparees) [reply architecture/core_concepts/containers_and_images.adoc]: reverted.

[the remark this answers is not in the document]
Comment (miminar) [reply architecture/core_concepts/containers_and_images.adoc]: Yes, I also think it's an unfortunate and misleading naming. But the change should be done globally in a dedicated PR.

**Comment by bparees** [comment]
@openshift/team-documentation ptal/merge.

applicable to however far back we have documentation for these features.

**Comment by bmcelvee** [comment]
LGTM

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../admin_guide/image_policy.adoc#admin-guide-image-policy[Image Policy]
|Updated example notes in the xref:../admin_guide/image_policy.adoc#image-policy-configuring-the-image-policy-admission-plug-in[Configuring the ImagePolicy Admission Plug-in] section.
%
|xref:../dev_guide/managing_images.adoc#dev-guide-managing-images[Managing Images]
|Added content for image streams lookup and resolution policies.
%
[rev_history]
|xref:../install_config/registry/extended_registry_configuration.adoc#install-config-registry-extended-configuration[Setting up the Registry -> Extended Registry Configuration]
|Added more content to the  xref:../install_config/registry/extended_registry_configuration.adoc#middleware-repository-pullthrough[Image Pullthrough] section.
%

**Comment by vikram-redhat** [comment]
Hey @bmcelvee - this needed to get picked to dedicated-3.6 and online-3.6 stage branches as well as per the labels and the reference table.

**Pull request merged**